### DISCUSSION TITLE
Panel under a location with nginx

### DISCUSSION THREAD

**Discussion by AngelBePro** [pterodactyl/panel#5196] [Support] [answered]
I tried a lot of things but nothing worked and the only way I managed was the intended way using the root of the domain. What I want is to use it under example.com/panel. I tried changing the .env but it gives me a white page without a body. I really suck at nginx configs so I would greatly appreciate it If someone made a config.

**Comment by rowansc1** [comment, marked as answer]
@AngelBePro - You could perhaps get this working by changing your webservers configuration. However, doing this isn't really supported by Pterodactyl so you'll need to figure that out yourself. Pterodactyl is designed to have its own dedicated FQDN, so you should really be doing that. 

Feel free to join the discord: https://discord.gg/pterodactyl
